api-reference/10 UI Widgets/dxDataGrid/1 Configuration/onEditorPreparing.md

```diff
@@ -54,13 +54,13 @@ The [properties](/api-reference/10%20UI%20Widgets/dxDataGrid/6%20Row '/Documenta
 Indicates whether the editor uses right-to-left representation.
 
 ##### field(e.setValue(newValue, newText)): any
-A method you should call to change the cell value and, optionally, the displayed value after the editor's value is changed.
+Changes the cell value and, optionally, the displayed value after the editor's value is changed.
```

Comment:
I think that the old version is better. Seeing that other fields are properties, we should underline that this one is a method.